app/src/main/res/layout/activity_main.xml

```diff
@@ -75,8 +75,27 @@
             android:layout_weight="1"/>
 
         <Button
```

Comment:
I believe it would be better to add this button separately from the two Share to FB Reels Button. Maybe add it before this whole button section, as a full width button like "Upload Video". Adding the three buttons side by side makes it hard to understand and breaks the text.